api/v2.0/swagger.yaml

```diff
@@ -2351,160 +2351,6 @@ paths:
           $ref: '#/responses/404'
         '500':
           $ref: '#/responses/500'
-  /projects/{project_name_or_id}/robots:
-    get:
-      summary: Get all robot accounts of specified project
-      description: Get all robot accounts of specified project
-      parameters:
-        - $ref: '#/parameters/requestId'
-        - $ref: '#/parameters/isResourceName'
-        - $ref: '#/parameters/projectNameOrId'
-        - $ref: '#/parameters/page'
-        - $ref: '#/parameters/pageSize'
-        - $ref: '#/parameters/query'
-        - $ref: '#/parameters/sort'
-      tags:
-        - robotv1
```

Comment:
you cannot remove those API since they are actually for the robot CRUD, but update the tag to robot.

Reply:
I am by no mean experienced enough with the code base but isn't the `/robots` endpoint what should be used for that ?

The corresponding V1 operations are also removed from the backend code.

Reply:
These APIs are using for robotv2 now. Since we are removing robotv1, let's update the tag to simply "robot" to reduce confusion.

Reply:
oh, it's ok to remove those APIs since we're using /robots for both project and system level robot.